transactions.go

```diff
@@ -278,6 +278,24 @@ func (mrr *MapReadResult) Counter(key Key) (val int32, err error) {
 	return 0, fmt.Errorf("counter entry with key '%s' not found", key)
 }
 
+// MapEntry represents the key and type of a map entry (embedded CRDT).
+type MapEntry struct {
+	Key      []byte
+	CrdtType CRDTType
+}
+
+// ListMapEntries gives access to the keys and types of map entries (embedded CRDTs).
+func (mrr *MapReadResult) ListMapEntries() (val []MapEntry, err error) {
```

Comment:
Usually an entry consists of key and value -- at least that's what I know from Java's API etc. 

As `ListMapEntries` only returns the keys, maybe `ListMapKeys` would be a better name?

Reply:
Indeed `ListMapKeys` is a better name for indicating that the return value does not include entry values.
I had initially used the name `ListMapKeys` but changed it thinking that the name does not include CRDT types that are also returned, but we can consider that an entry key consists of key and a CRDT type.